tests/jest/application-navigation/ApplicationNavigation.test.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+import ApplicationNavigation from '../../../src/application-navigation/ApplicationNavigation';
+
+describe('ApplicationNavigation', () => {
+  it('should render default element', () => {
+    const shallowComponent = shallow(
+      <ApplicationNavigation />,
+    );
+
+    expect(shallowComponent).toMatchSnapshot();
+  });
+
+  it('should render with all props', () => {
+    const shallowComponent = shallow(
+      <ApplicationNavigation
+        activeBreakpoint="large"
+        titleConfig={{
+          title: 'test-title',
+        }}
+        activeNavigationItemKey="my-test-key"
+        hero={<div>my test hero</div>}
+        userConfig={{
+          name: 'user-name',
+          detail: 'user-detail',
+          initials: 'user-initials',
+          imageSrc: 'user-src',
+        }}
+        navigationItems={[{ text: 'test-text', key: 'my-test-key' }]}
+        extensionItems={[{ text: 'test-text-1', key: 'my-test-key-1', icon: <span>my icon</span> }]}
+        utilityItems={[{ text: 'test-text-2', key: 'my-test-key-2', icon: <span>my icon</span> }]}
+        notifications={{ 'my-test-key-1': 3 }}
+        navigationRenderFunction={jest.fn()}
+        onSelectNavigationItem={jest.fn()}
+        onSelectExtensionItem={jest.fn()}
+        onSelectUtilityItem={jest.fn()}
+        onSelectSettings={jest.fn()}
+        onSelectHelp={jest.fn()}
+        onSelectLogout={jest.fn()}
+        onDrawerMenuStateChange={jest.fn()}
+      >
+        <div>test child content</div>
+      </ApplicationNavigation>,
+    );
+
+    expect(shallowComponent).toMatchSnapshot();
+  });
+
+  it('should render with all props', () => {
```

Comment:
looks like this is a duplicate of [the test before this](https://github.com/cerner/terra-application/pull/5/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.md&file-filters%5B%5D=.png&file-filters%5B%5D=.snap#diff-611946ada81ee56923728d2306a6ee54R14)